test/t8_test_cmesh_copy.c

```diff
@@ -24,8 +24,18 @@
 #include "t8_cmesh/t8_cmesh_trees.h"
 #include "t8_cmesh/t8_cmesh_partition.h"
 
+
+sc_MPI_Comm  comm_list[]={sc_MPI_COMM_WORLD,sc_MPI_COMM_NULL,sc_MPI_COMM_SELF};
+size_t num_comm = sizeof(comm_list) / sizeof(comm_list[0]); 
```

Comment:
I know that this works, but we should avoid this syntax. The size of static arrays should be known at compile time.

```suggestion
#define T8_CMESH_TEST_NUM_COMMS 3
sc_MPI_Comm  comm_list[T8_CMESH_TEST_NUM_COMMS]={sc_MPI_COMM_WORLD,sc_MPI_COMM_NULL,sc_MPI_COMM_SELF};
```

Reply:
If you need global variables (such as the comm_list) they should have a  `t8_` prefix in their name.
(Usually we should avoid global variables, in this case it is necessary)